eahub/profiles/admin.py

```diff
@@ -30,7 +34,7 @@ def queryset(self, request, queryset):
 
 @admin.register(Profile)
 class ProfileAdmin(admin.ModelAdmin, utils.ExportCsvMixin):
-    actions = ["export_csv", "approve_profiles"]
+    actions = ["export_csv", "approve_profiles", "delete_profiles_and_users"]
```

Comment:
deletion doesn't seem to work. I've tried to delete my test user, but when I come back to the profile admin panel it's still there:
![image](https://user-images.githubusercontent.com/15908208/106391252-584bfb80-63e4-11eb-9694-ebf394f55afb.png)

Reply:
ok thanks, I'll check